python/doc/theory/reliability_sensitivity/sensitivity_form.rst

```diff
@@ -4,61 +4,60 @@ Sensitivity Factors from FORM method
 ------------------------------------
 
 | Sensitivity Factors are evaluated under the following context:
-  :math:`\vect{X}` denotes a random input vector, representing the
+  :math:`\inputRV` denotes a random input vector, representing the
   sources of uncertainties, :math:`\pdf` its joint density probability,
-  :math:`\vect{d}` a deterministic vector, representing the fixed
-  variables :math:`g(\vect{X}\,,\,\vect{d})` the limit state function of
+  :math:`\model` the limit state function of
   the model,
-  :math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-  the event considered here and :math:`{g(\vect{X}\,,\,\vect{d}) = 0}`
+  :math:`\cD_f = \{\vect{x} \in \Rset^\inputDim \, / \, \model(\vect{x}) \le 0\}`
+  the event considered here and :math:`{\model(\inputRV) = 0}`
   its boundary (also called limit state surface).
 | The probability content of the event :math:`\cD_f` is :math:`P_f`:
 
   .. math::
     :label: PfX11
 
-    P_f = \int_{g(\vect{X}\,,\,\vect{d}) \le 0}  \pdf\, d\vect{x}.
+    P_f = \int_{\model(\inputRV) \le 0}  \pdf\, d\vect{x}.
 
 | In this context, the probability :math:`P_f` can often be
   efficiently estimated by FORM or SORM approximations.
 | The FORM importance factors offer a way to analyze the sensitivity of
   the probability the realization of the event with respect to the
-  parameters of the probability distribution of :math:`\vect{X}`.
+  parameters of the probability distribution of :math:`\inputRV`.
 
 | A sensitivity factor is defined as the derivative of the Hasofer-Lind
   reliability index with respect to the parameter :math:`\theta`. The
   parameter :math:`\theta` is a parameter in a distribution of the
-  random vector :math:`\vect{X}`.
+  random vector :math:`\inputRV`.
 | If :math:`\vect{\theta}` represents the vector of all the parameters
-  of the distribution of :math:`\vect{X}` which appear in the definition
+  of the distribution of :math:`\inputRV` which appear in the definition
   of the isoprobabilistic transformation :math:`T`, and
-  :math:`U_{\vect{\theta}}^{*}` the design point associated to the event
+  :math:`\vect{U}_{\vect{\theta}}^{*}` the design point associated to the event
   considered in the :math:`U`-space, and if the mapping of the limit
   state function by the :math:`T` is noted
-  :math:`h(\vect{U}\,,\,\vect{\theta}) =  g[T^{-1}(\vect{U}\,,\,\vect{\theta}), \vect{d}]`,
+  :math:`h(\vect{U}\,,\,\vect{\theta}) =  \model[T^{-1}(\vect{U}\,,\,\vect{\theta})]`,
   then the sensitivity factors vector is defined as:
 
   .. math::
 
-      \nabla_{\vect{\theta}} \beta_{HL} =  \displaystyle +\frac{1}{||\nabla_{\vect{\theta}} h(U_{\vect{\theta}}^{*}, \vect{d})||} \nabla_{\vect{u}} G(U_{\vect{\theta}}^{*}, \vect{d}).
+      \nabla_{\vect{\theta}} \beta_{HL} =  \displaystyle +\frac{1}{||\nabla_{\vect{\theta}} h(U_{\vect{\theta}}^{*})||} \nabla_{\vect{u}} G(U_{\vect{\theta}}^{*}).
```

Comment:
```suggestion
      \nabla_{\vect{\theta}} \beta_{HL} =  \displaystyle +\frac{1}{||\nabla_{\vect{\theta}} h(U_{\vect{\theta}}^{*})||} \nabla_{\vect{u}} \model(U_{\vect{\theta}}^{*}).
```

Reply:
No: G here is h defined as $h = g \circ T^{-}$. It is not $g$.